### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by pedropbr**
Fixed issues
- https://contentful.slack.com/archives/pm/p1469706161000007
- https://github.com/contentful/slash-developers/issues/285
- https://github.com/contentful/slash-developers/issues/197

### CODE REVIEW

File: apiary/cda.apib [hunk 1]
```diff
@@ -20,6 +20,7 @@ As an authorization HTTP request-header field and as an URI query parameter name
 
 ## API rate limits
 
+Rate limits refer to the amount of requests per unity of time that can be done against our APIs.
```
Comment (realityking) [inline apiary/cda.apib]: _unit_ of time

**Comment by realityking** [comment]
From the linked ticket #285 the documentation for single API key endpoint is not included here.

The part about the rate limiting should probably be reviewed by someone involved in that discussion. I'm not sure if it's enough to get the point across.

**Comment by pedropbr** [comment]
@realityking we can leave the #285 open, no problem

**Comment by pedropbr** [comment]
@realityking typo fixed!

**Comment by realityking** [comment]
LGTM

**Pull request merged**